examples/vecbuiltin/vecbuiltin_exec.c

```diff
@@ -0,0 +1,84 @@
+#include "poclu.h"
+#include <CL/opencl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+  int
```

Comment:
check indent

Reply:
do we need extern "C" is this for Windows?

Reply:
I think it's not strictly required, i can remove it.